Assets/SEE/Tools/FaceCam/FaceCam.cs

```diff
@@ -253,6 +257,36 @@ public override void OnNetworkSpawn()
             base.OnNetworkSpawn();
         }
 
+        /// <summary>
+        /// Initializes <see cref="webCamTextureToMatHelper"/> if not already set.
+        /// </summary>
+        private void Initialize()
+        {
+            // For dynamically spawned NetworkObjects (instantiating a network Prefab
+            // during runtime) the OnNetworkSpawn method is invoked before the Start
+            // method is invoked. So, it's important to be aware of this because finding
+            // and assigning components to a local property within the Start method exclusively
+            // will result in that property not being set in a NetworkBehaviour component's
+            // OnNetworkSpawn method when the NetworkObject is dynamically spawned. To
+            // circumvent this issue, you can have a common method that initializes the
+            // components and is invoked both during the Start method and the
+            // OnNetworkSpawned method. That's the purpose of this method.
+
+            if (webCamTextureToMatHelper == null)
+            {
+                if (!gameObject.TryGetComponentOrLog(out webCamTextureToMatHelper))
+                {
+                    enabled = false;
+                }
+            }
+
+            if (!gameObject.TryGetComponentOrLog(out meshRenderer))
+            {
+                gameObject.SetActive(false);
+                return;
```

Comment:
This line seems unnecessary.
```suggestion
```

Reply:
True, it's not needed, but what if anyone adds a line after the if statement at a later point in time?